python3/CHANGELOG.md

```diff
@@ -1,5 +1,89 @@
 # IBM Functions Python 3 Runtime Container
 
+## 1.4.0
+Changes:
+- updated httplib2 (0.11.3)
```

Comment:
Changes section:
Can you add as the first bullet that python version was updated, from previous version to new version?